phpunit-bootstrap.php

```diff
@@ -5,8 +5,27 @@
 use Brick\Geo\Engine\PDOEngine;
 use Brick\Geo\Engine\SQLite3Engine;
 use Brick\Geo\Engine\GEOSEngine;
+use Doctrine\DBAL\Types\Type;
+
+/** @var \Composer\Autoload\ClassLoader $classLoader */
+$classLoader = require 'vendor/autoload.php';
+
+//Add namespace for doctrine base tests
+$classLoader->addPsr4('Doctrine\\Tests\\', [
+    __DIR__ . '/vendor/doctrine/orm/tests/Doctrine/Tests',
+    __DIR__ . '/vendor/doctrine/dbal/tests/Doctrine/Tests'
+]);
+$classLoader->loadClass('Doctrine\Tests\DbalFunctionalTestCase');
+$classLoader->loadClass('Doctrine\Tests\DBAL\Mocks\MockPlatform');
+
+Type::addType('geometry', 'Brick\Geo\Doctrine\Types\GeometryType');
+Type::addType('linestring', 'Brick\Geo\Doctrine\Types\LineStringType');
+Type::addType('multilinestring', 'Brick\Geo\Doctrine\Types\MultiLineStringType');
+Type::addType('multipoint', 'Brick\Geo\Doctrine\Types\MultiPointType');
+Type::addType('multipolygon', 'Brick\Geo\Doctrine\Types\MultiPolygonType');
+Type::addType('point', 'Brick\Geo\Doctrine\Types\PointType');
+Type::addType('polygon', 'Brick\Geo\Doctrine\Types\PolygonType');
```

Comment:
Could you please use PHP 5.5's `::class` syntax here: `use Brick\Geo\Doctrine\Types;` at the top of the file, then `addType('...', Types\GeometryType::class);`.